packages/shared/src/components/post/write/CreatePostButton.tsx

```diff
@@ -72,11 +87,38 @@ export function CreatePostButton<Tag extends AllowedTags>({
   const shouldShowAsCompact =
     compact !== false && ((isLaptop && !isLaptopL) || compact);
 
+  const getTooltipContent = () => {
+    if (!shouldShowPollTooltip) {
+      return 'New Post';
+    }
+
+    return (
+      <div className="flex flex-col gap-2 py-1">
+        <Typography type={TypographyType.Subhead} center>
+          You can now create polls!
+        </Typography>
+        <Button
+          variant={ButtonVariant.Secondary}
+          size={ButtonSize.Small}
+          className="bg-surface-invert"
+          tag="a"
+          href={`${link.post.create}?poll=true`}
+        >
+          Try it now!
+        </Button>
+      </div>
+    );
+  };
+
   return (
     <ConditionalWrapper
-      condition={shouldShowAsCompact}
+      condition={shouldShowAsCompact || shouldShowPollTooltip}
       wrapper={(component: ReactElement) => (
-        <Tooltip side="bottom" content="New Post">
+        <Tooltip
+          side="right"
+          content={getTooltipContent()}
+          alwaysOpen={shouldShowPollTooltip}
```

Comment:
doesn't open. alone suffice? I was kinda happy to finally get rid of that alwaysOpen state as well.

Reply:
<img width="492" height="113" alt="Screenshot 2025-09-17 at 7 49 39 PM" src="https://github.com/user-attachments/assets/523eb628-e994-48c7-961c-fe86d969aa06" />

There seem to be no open property? The `open` value is handled within the component itself.

Reply:
Do you mean to expose that `open` property instead?

Reply:
Yeah, I agree, basically a prop for controlled state.

Reply:
Pushed!

Although we can't use `useState(open)` since the state whether to display it or not is determined after render.